### PR TITLE
And13 ipp conflicts

### PR DESCRIPTION
**Pull request by amritaintel**

### CODE REVIEW

File: caas/device.mk [hunk 1]
```diff
@@ -584,6 +584,13 @@ PRODUCT_COPY_FILES += \
         $(INTEL_PATH_COMMON)/touch/Vendor_2386_Product_3115.idc:system/usr/idc/Vendor_2386_Product_3115.idc\
         $(INTEL_PATH_COMMON)/touch/Vendor_056a_Product_489c.idc:system/usr/idc/Vendor_056a_Product_489c.idc
 ##############################################################
+# Source: device/intel/mixins/groups/googleservice/gms/product.mk
```
Comment (yyao3) [inline caas/device.mk:587]: Why this one is related with IPP conflicts?

File: caas/mixins.spec [hunk 2]
```diff
@@ -40,6 +40,7 @@ midi: true
 touch: cvt0f21
 navigationbar: true
 device-type: tablet
+googleservice: default
```
Comment (yyao3) [inline caas/mixins.spec:43]: Same here.

**Review by yyao3 — COMMENTED** [review]
GMS is not related with NN?

**Review by yyao3 — APPROVED** [review]

**Review by yyao3 — APPROVED** [review]

**Comment by amritaintel** [comment]
Had to revert an existing PR for disabling GMS

**Pull request closed without merging**